src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +94,57 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page
+     *
+     * @param propertiesFiles
+     *            property files to read
+     */
+    public void parsePropertiesFiles(List<String> propertiesFiles) {
+
+        if (propertiesFiles.isEmpty()) {
+
+            return;
+
+        }
+
+        for (String propertyFile : propertiesFiles) {
+
+            LOG.info("Processing MetadataFile - %s", propertyFile);
```

Comment:
don't think it gives any value to have such message here

Reply:
ok will remove